next-app/src/components/administration/tokens/Airdrop.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Card,
+  CardHeader,
+  CardTitle,
+  CardDescription,
+  CardContent,
+  CardFooter,
+} from "@/components/ui/card";
+import { Label } from "@/components/ui/label";
+import { Input } from "@/components/ui/input";
+import { Button } from "@/components/ui/button";
+import { JSX, SVGProps, useState } from "react";
+
+const Airdrop = () => {
+  const [address, setAddress] = useState("");
+  const [error, setError] = useState("");
+  const [showConfirmation, setShowConfirmation] = useState(false);
+  const handleSaveChanges = () => {
```

Comment:
Duplicated hook

Reply:
Duplicated is the good word ?? I don't get it